docs_user/modules/proc_adopting-the-loadbalancer-service.adoc

```diff
@@ -5,39 +5,252 @@
 During the adoption process the {loadbalancer_first_ref} service
 must stay disabled in the new control plane.
 
-== Certificates
-Before running the script below the shell variables `CONTROLLER1_SSH` and
-`CONTROLLER1_SCP` must be set to contain the command to log into one of the
-controllers using `ssh` and `scp` respectively as root user as shown below.
+== Control plane adoption steps
 
+. Create an alias for the `openstack` command
++
 [source,bash]
 ----
-$ CONTROLLER1_SSH="ssh -i <path to the ssh key> root@192.168.122.100"
-$ CONTROLLER1_SCP="scp -i <path to the ssh key> root@192.168.122.100"
+$ alias openstack="oc exec -t openstackclient -- openstack"
 ----
-
-Make sure to replace `<path to the ssh key>` with the correct path to the ssh
-key for connecting to the controller.
-
+. Certificates
++
+Before running the script below set the
+`CONTROLLER1_SCP` shell variable based on the value of the
+existing `CONTROLLER1_SSH` variable:
++
+[source,bash]
+----
+$ CONTROLLER1_SCP=$(echo "$CONTROLLER1_SSH" | sed 's/^ssh/scp/g')
+----
++
+Then run the following set of commands in order to regenerate the
+keys and certificates and install the data in OpenShift. These
+commands convert the existing single CA configuration into a
+dual CA configuration.
++
 [source,bash]
 ----
 include::../../tests/roles/dataplane_adoption/tasks/octavia_certs.yaml[lines="7..83",indent=0]
 ----
 
-These commands convert the existing single CA configuration into a dual CA configuration.
-
-// == TODO: Other things
-// TODO
+. Add the octavia interfaces to each NodeNetworkConfigurationPolicy.
++
+The following command adds the network interface that is being
+used as the VLAN base interface for the management network
+(in this example `enp6s0`) for network isolation on the OpenShift nodes.
+Please replace `enp6s0` with the name of the network interface in your
+OpenShift setup.
```

Comment:
You might want to add a comment that the mtu value is only an example and may differ depending on the requirments (or something like that)

Reply:
Thanks. Added.